modules/custom/d_media/d_media.module

```diff
@@ -24,3 +24,15 @@ function d_media_theme() {
     ],
   ];
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function d_media_form_editor_image_dialog_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+  // Allow upload svg file in ckeditor, if config wysiwyg_upload_svg_disable is null or false.
+  $config = \Drupal::config('d_media.settings');
+  $svg_upload_disabled = isset($config) ? $config->get("wysiwyg_upload_svg_disable") : NULL;
```

Comment:
Gdzie ustawiamy tę zmienną `wysiwyg_upload_svg_disable`?

Reply:
nigdzie ;P jak ktoś bardzo chciał to sobie ogarnie, zeby nie musial hakowac.

Reply:
ee, trochę dużo kombinowania, dałbym to z defaultu bez zmiennej, jeśli ktoś będzie bardzo chciał to sobie tego altera wyłączy

Reply:
no spoko, to wywalam